app/src/main/java/com/celzero/bravedns/util/HttpRequestHelper.kt

```diff
@@ -117,10 +118,9 @@ class HttpRequestHelper{
                 DownloadManager.STATUS_RUNNING -> statusText = "STATUS_RUNNING"
                 DownloadManager.STATUS_SUCCESSFUL -> {
                     statusText = "STATUS_SUCCESSFUL"
-                    //reasonText = "Filename:\n$filename"
                 }
             }
-            if(DEBUG) Log.d(LOG_TAG,"Reason: $reasonText")
+            if(DEBUG) Log.d(LOG_TAG_DOWNLOAD,"Reason: $reasonText")
```

Comment:
run code formatter. Do not submit for code-review without formatting the code. This is basic.

Reply:
Ran code-formatter for all the files.